_test/test_sqw/test_PixelData.m

```diff
@@ -545,6 +545,16 @@ function test_constructing_from_PixelData_with_valid_file_inits_faccess(obj)
         assertTrue(new_pix.has_more());
     end
 
+    function test_num_pixels_is_a_double_if_faccess_returns_uint(obj)
```

Comment:
You need tests confirming that num_pix is double in the three use-cases:
- faccess backed (<1 page)
- in-memory backed (e.g. array created object)
- file-backed contained less than one-page of data